cmd/k8s-operator/connector.go

```diff
@@ -59,15 +58,21 @@ type ConnectorReconciler struct {
 
 	mu sync.Mutex // protects following
 
-	connectors set.Slice[types.UID] // for connectors gauge
+	connectors    set.Slice[types.UID] // for connectors gauge
+	subnetRouters set.Slice[types.UID] // for subnet routers gauge
+	exitNodes     set.Slice[types.UID] // for exit nodes gauge
```

Comment:
> the total number of elements added, not just the number of unique ones.

What do you mean? We specifically want to avoid counting the connector multiple times as reconcilers will run many times in response to cluster events, but we want to count how many connectors we have, not how many times a connector was reconciled

Reply:
Sorry, please disregard this. I got confused a bit while reading the code of `set.Set` and `set.Slice`. I suspect the latter is only necessary if you need to preserve order, but `Len()` does count unique elements in both cases.